src/use-swr.ts

```diff
@@ -645,7 +641,7 @@ function useSWR<Data = any, Error = any>(
     (newData, shouldRevalidate) => {
       return internalMutate(cache, keyRef.current, newData, shouldRevalidate)
     },
-    []
+    [cache]
```

Comment:
Do you think if we should add ESLint ignore, since we’re sure that cache is not gonna change?